.github/ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] General guidance or best practices
+- [ ] Usability - when to use
+- [ ] Usability - when to consider something else
+- [ ] Accessibility considerations
+
+## Validate HTML
+If you are using Google Chrome or Firefox consider getting the [axe plugin](https://www.deque.com/axe) for validation.
+**What to check:**
+- If testing on W3.org
+  - [ ] Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+- If testing with axe plugin
```

Comment:
Same thought here. I'd like to make this actionable as the second item, maybe `Test with aXe plugin` for the bullet point, and then your excellent checkbox point below it.